protobuf-maven-plugin/src/it/protoc-descriptor-file/src/test/java/org/example/helloworld/ProtobufTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2023 - 2024, Ashley Scopes.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.example.helloworld;
```

Comment:
Think it is safe to just remove this test class, I don't think we need to worry about this too much as it is tested elsewhere already. That means we can remove junit-jupiter as well in this invoker test.

We can always check that the Java files are still output in the test.groovy though if you like?